### PR TITLE
cloudtasks: initial commit

### PR DESCRIPTION
**Pull request by broady**
Note: you must be whitelisted to use this API.

### CODE REVIEW

File: cloudtasks/flex_app/main.go [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+		}
+	}
+
+	datastoreClient, err = datastore.NewClient(ctx, projectID)
+	if err != nil {
+		log.Fatalf("Could not initialize datastore client: %v", err)
+	}
+
+	http.HandleFunc("/", rootHandle)
+	http.HandleFunc("/payload", payloadHandle)
```
Comment (steren) [inline cloudtasks/flex_app/main.go]: We performed user studies and there was a bit misunderstanding about the string "log_payload".
Please replace "payload" with "example_task_handler" in this sample.
The Python master samples have been changed: https://github.com/GoogleCloudPlatform/python-docs-samples/pull/1288/files

Comment (broady) [reply cloudtasks/flex_app/main.go]: do we need the word "example"? why not just "task_handler"

**Comment by grayside** [comment]
Taking this one over per discussion with @broady

**Comment by grayside** [comment]
This PR is deprecated

**Pull request closed without merging**